install-dev/init.php

```diff
@@ -30,6 +30,12 @@
     die('You need at least PHP 5.4 to run PrestaShop. Your current PHP version is '.PHP_VERSION);
 }
 
+/* in dev mode - check if composer was executed */
+if (is_dir(_PS_ROOT_DIR_.DIRECTORY_SEPARATOR.'admin-dev') && (!is_dir(_PS_ROOT_DIR_.DIRECTORY_SEPARATOR.'vendor') ||
```

Comment:
![I like to live dangerously](http://www.socialmemegenerator.com/wp-content/uploads/2015/10/i-like-to-live-dangerously_b6e28243d11.png)